app/__tests__/popup.test.jsx

```diff
@@ -113,6 +112,7 @@ describe("Popup", () => {
       expect(localProps.errorNotify).toHaveBeenCalledWith(
         "Cannot Start Listening on chrome://version"
       );
+      expect(localProps.enabled).toEqual(false)
```

Comment:
instead of this **a better test** will be to make sure we are not calling `disableLogging` or `getRequests`